cmd/manager/main.go

```diff
@@ -120,6 +122,11 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err := openshiftv1.AddToScheme(mgrScheme); err != nil {
+		log.Error(err, "Failed adding openshift/v1 to scheme")
+		os.Exit(1)
+	}
```

Comment:
Why is this required?
I see it is also being used in multiple tests.

I never understood the purpose of this.

Reply:
If I remember correctly, this is because we need to be able to understand and manipulate the specs for all the API objects. Since Templates are an OpenShift construct, we need to add the entire openshift scheme.